src/main/java/uk/gov/ons/census/casesvc/service/EventService.java

```diff
@@ -18,6 +20,9 @@
 public class EventService {
 
   public static final String CREATE_CASE_SAMPLE_RECEIVED = "Create case sample received";
+  public static final String CREATE_BULK_CASE_SAMPLE_RECEIVED = "Create bulk case sample received";
+  public static final String BULK_PROCESSING_EVENT_CHANNEL = "AR";
+  public static final String BULK_PROCESSING_EVENT_SOURCE = "RM";
```

Comment:
these things are only public for the convenience of the tests, which is very bad practice. Should be private

Reply:
they are just constants though so I guess it's not the end of the world, but it's a bad habit to get into